docs/config.md

```diff
@@ -39,12 +48,10 @@ easier to write down in an unconfusing way. If you are confident with TOML
 then use whichever suits you in your config. If you mix dotted keys and headings,
 **put the dotted keys before the first heading**.
 
-The other thing to remember is that the value of a setting (the part to the
-right of the `=` sign) should be surrounded in quotes if it's a string. That's
+That's
 probably enough TOML to keep you out of trouble but the [syntax guide] is very
 short if you ever need to check.
```

Comment:
nit: The new text looks weirdly wrapped with just "That's" on a single line. Could you rewrap it?